starcheck/src/starcheck

```diff
@@ -3,12 +3,12 @@
 unset LD_LIBRARY_PATH
 unset SYBASE
 unset PYTHONPATH
-SKA_bin=`dirname $0`
-eval `${SKA_bin}/flt_envs -shell sh`
+eval `${SKA}/bin/flt_envs -shell sh`
```

Comment:
I don't know if we really needed the script to figure out the path of flt_envs using dirname.  Seems unnecessary (and may or may not be correct, depending on where we decide to put the `starcheck` wrapper).